### PR TITLE
feature/Binny-10233-CreatePIIFilter

### PR DESCRIPTION
**Pull request by dumathane**
## Screenshots/Video

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;

## Testing

- [ ] Tested on iOS 
- [ ] Tested on Android 

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [ ] PR is connected to issue(s)
- [ ] Tests are included to cover this change (when possible)
- [ ] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [ ] No secrets or API keys are checked in
- [ ] All imports are absolute (no relative imports)
- [ ] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

File: VAMobile/src/constants/common.ts [hunk 1]
```diff
@@ -57,6 +57,9 @@ export const MAIL_TO_REGEX_EXP = new RegExp(
 export const PHONE_REGEX_EXP = new RegExp(
   /^\s*(?:\+?(\d{0,3}))?[-. (]*(\d{3})[-. )]*(\d{3})[-. ]*(\d{4})(?: *(x)(\d+))?,?.?\s*$/,
 )
+
+export const SSN_REGEX_EXP = new RegExp(/^\d{3}-?\d{2}-?\d{4}$/)
```
Comment (Sparowhawk) [inline VAMobile/src/constants/common.ts:61]: export const SSN_REGEX_EXP = new RegExp(/^\d{3}-?\s?\d{2}-?\s?\d{4}$/)

Comment (Sparowhawk) [reply VAMobile/src/constants/common.ts:61]: the regex to include searching for spaces is this

Comment (Sparowhawk) [inline VAMobile/src/constants/common.ts:61]: export const SSN_REGEX_EXP = new RegExp(/^\d{3}-?\s?\d{2}-?\s?\d{4}$/)

Comment (Sparowhawk) [reply VAMobile/src/constants/common.ts:61]: this is the regex for those changes to include spaces as part of the regex search

File: VAMobile/src/screens/HomeScreen/ProfileScreen/SettingsScreen/InAppFeedbackScreen/InAppFeedbackScreen.tsx [hunk 2]
```diff
@@ -28,8 +29,28 @@ function InAppFeedbackScreen({ navigation }: InAppFeedbackScreenProps) {
   // })
 
   const onSubmit = (): void => {
+    const { found, newText } = checkStringForPII(task)
     // logAnalyticsEvent(Events.vama_feedback_submitted(taskCompleted, satisfaction))
-    navigation.goBack()
+    if (found) {
+      console.log('Before task: ', task)
```
Comment (Sparowhawk) [inline VAMobile/src/screens/HomeScreen/ProfileScreen/SettingsScreen/InAppFeedbackScreen/InAppFeedbackScreen.tsx:35]: probably should clean up these console logs

**Review by Sparowhawk — CHANGES_REQUESTED** [review]

File: VAMobile/src/constants/common.ts (hunk 1 above)
Comment (dumathane) [reply VAMobile/src/constants/common.ts:61]: I'm actually going to just make these both strict '-' searches currently and we can spin off more in-depth PII filtering at a later date if it's necessary.

File: VAMobile/src/screens/HomeScreen/ProfileScreen/SettingsScreen/InAppFeedbackScreen/InAppFeedbackScreen.tsx (hunk 2 above)
Comment (dumathane) [reply VAMobile/src/screens/HomeScreen/ProfileScreen/SettingsScreen/InAppFeedbackScreen/InAppFeedbackScreen.tsx]: fixed as suggested.

**Review by cadibemma — APPROVED** [review]

**Review by rbontrager — APPROVED** [review]

**Pull request merged**